installer/mod.ts

```diff
@@ -153,6 +153,7 @@ async function generateExecutable(
   filePath: string,
   commands: string[]
 ): Promise<void> {
+  commands = commands.map((v): string => '"' + v + '"');
```

Comment:
Do these strings need to be escaped too? What if they contain a quote mark?

Reply:
yes. I did it.